### DISCUSSION TITLE
最新内测更新后，X11模式下无法使用Super键作为快捷键

### DISCUSSION THREAD

**Discussion by debuggerx01** [linuxdeepin/developer-center#6083] [Troubleshooting | 故障排查] [answered]
更新最新的内测后：
- X11模式下无法通过单击Super键激活启动器
- 修改快捷键为其他，如Super+Z可以正常激活启动器
- 修改其他任何命令的快捷键为Super后，操作无效
- Wayland下无此现象，可以正常使用单独的Super作为快捷键触发

猜测可能与 @dengbo11 最近的修改有关，但简单看代码变更没找到具体原因，烦请检查一下谢谢！

**Comment by BLumia** [comment, marked as answer]
https://github.com/linuxdeepin/developer-center/issues/6084 看看是不是已经在这个里解决了？

**Reply by debuggerx01** [reply]
嗯，安装测试包后问题解决
